app/src/main/java/com/example/pygmyhippo/user/EventFragment.java

```diff
@@ -90,57 +191,56 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         eventAboutDescriptionView.setText(event.getDescription());
     }
 
-    @Override
```

Comment:
just curious, why did you remove oncreateview here?

Reply:
OCD. Its still in EventFragment just at a different spot in the file.